modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -1928,6 +1928,61 @@ definition {
 		ObjectAdmin.assertRequiredErrorPresent();
 	}
 
+	@description = "Verify if an error message appears when trying to save an Update an object Entry action with a Standalone trigger when using the invalid expression"
+	@priority = 4
+	test CannotSaveUpdateAnObjectEntryActions {
+		task ("Given: An object with a standalone action created") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object",
+				objectName = "CustomObject",
+				pluralLabelName = "Custom Objects");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Integer",
+				fieldLabelName = "Custom Integer",
+				fieldName = "customInteger",
+				fieldType = "Integer",
+				isRequired = "true",
+				objectName = "CustomObject");
+		}
+
+		task ("When: The user adds an invalid expression to the Update an Object Entry action") {
```

Comment:
This block isn't adding the expression yet. Once you remove the assertStandaloneUpdateEntryError macro it should make more sense since though since the fillObjectAction step is tied to it and can be moved.